Trustroots/trustroots
# Change all occurences of `withTranslation` to `useTranslation`

#### Proposed Changes

- change all occurences of `withTranslation` to `useTranslation`
- migrate class components to function components to be able to use the hook
- add `@/config/client/i18n` to a few places
- remove obsolete documentation

#### Why?

- #1291 will extract translations to their proper namespaces
- `withTranslation` were hanging around for historical reasons

#### Testing Instructions

- git switch `use-translation`
- browse things and see that they still work

_tip: hide whitespace changes when reviewing the code_

![hide whitespace changes](https://user-images.githubusercontent.com/7449720/76378835-cad86680-634e-11ea-996c-a9e8a6ca8bdf.png)
